core/src/main/resources/reference.conf

```diff
@@ -59,11 +59,34 @@ akka {
 
     # Note that without further configuration, all other actors run on the default dispatcher
   }
+
+  coordinated-shutdown.phases {
+    # This phase is used to give time to Cromwell to abort all workflows upon shutdown. It's only used if system.abort-jobs-on-terminate = true
+    abort-all-workflows {
+      # /!\ This depends-on field should NOT be modified
+      depends-on = [service-unbind]
+      # Adjust this timeout according to the maximum time you're ready to wait for to allow Cromwell to abort all workflows on shutdown
```

Comment:
I think this would read a little more doc-y minus the "you're ready to wait for" bit, but @mcovarr is the wordsmith here